src/SourceBuild/tarball/content/smoke-test.sh

```diff
@@ -55,7 +55,9 @@ excludeOnlineTests=false
 devCertsVersion="$DEV_CERTS_VERSION_DEFAULT"
 testingDir="$SCRIPT_ROOT/testing-smoke"
 cliDir="$testingDir/builtCli"
-logFile="$testingDir/smoke-test.log"
+logsDir="$testingDir/logs"
```

Comment:
The changes to this file are to collect all of the logs in a common folder.  This is convenient for collection purposes as well as it enhances the dev UX as they don't pollute the top level folder w/logs.